src/routes/transactions/helpers/swap-order.helper.ts

```diff
@@ -118,15 +122,44 @@ export class SwapOrderHelper {
     return url;
   }
 
+  /**
+   * Checks if the app associated with an order is allowed.
+   *
+   * @param order
```

Comment:
Nit: this is missing a description.

Reply:
Fixed in [4f1ee88](https://github.com/safe-global/safe-client-gateway/pull/1479/commits/4f1ee881de34211aba49803d17cffcf541c24e83)